packages/rich-text/src/component/index.native.js

```diff
@@ -1069,8 +1069,8 @@ export class RichText extends Component {
 		if ( this.needsSelectionUpdate ) {
 			this.needsSelectionUpdate = false;
 			selection = {
-				start: this.props.selectionStart,
-				end: this.props.selectionEnd,
+				start: this.props.selectionStart ?? 0,
+				end: this.props.selectionEnd ?? 0,
```

Comment:
I noticed that `this.props.selectionStart` and `this.props.selectionEnd` is also used in the following lines after this block:

https://github.com/WordPress/gutenberg/blob/2e95aaef7e3bd68e156f12c1a313a985e8536a3d/packages/rich-text/src/component/index.native.js#L1090

https://github.com/WordPress/gutenberg/blob/2e95aaef7e3bd68e156f12c1a313a985e8536a3d/packages/rich-text/src/component/index.native.js#L1094

If we want to ensure that both props are always defined, we might consider having them as constants within the `render` function, wdyt?